feg/gateway/services/session_proxy/servicers/session_controller_test.go

```diff
@@ -238,11 +158,10 @@ func TestStartSessionGyFail(t *testing.T) {
 		}
 	}).Once()
 
-	mockPolicyDb.On("GetChargingKeysForRules", mock.Anything, mock.Anything).Return(
+	mockPolicyDBClient.On("GetChargingKeysForRules", mock.Anything, mock.Anything).Return(
 		[]policydb.ChargingKey{{RatingGroup: 1}}, nil).Once()
 	// no omnipresent rules
-	mockPolicyDb.On("GetOmnipresentRules").Return([]string{}, []string{}).Once()
-	mockPolicyDb.On("GetRuleIDsForBaseNames", mock.Anything).Return([]string{}).Once()
```

Comment:
why this has disappeared?

Reply:
I added this `mockPolicyDBClient.AssertExpectations(t)` down at line 188 and this call was never called. We were never asserting on the expectations before.